packages/cli/src/PublicApi/v1/handlers/users/users.handler.ee.ts

```diff
@@ -68,4 +76,29 @@ export = {
 			});
 		},
 	],
+	createUser: [
+		globalScope('user:create'),
+		async (req: Create, res: Response) => {
+			const usersInvited = await Container.get(InvitationController).inviteUser(req);
```

Comment:
we should not be using private controllers in the public API. but I guess since all the payload validation is still in the controllers, we'd have to duplicate a lot of code.

Reply:
For now the tests will ensure we catch any unexpected changes to the Public API.